### DISCUSSION TITLE
Is there a way to extract graphql query in the resolver and convert it to a `GraphQLQueryRequest`?

### DISCUSSION THREAD

**Discussion by Rfank2021** [Netflix/dgs-framework#1177] [Q&A]
I didn't find a way to auto convert to a `GraphQLQueryRequest`. Maybe I can create a projection based on query. Or how can I projection all fields?

**Comment by srinivasankavitha** [comment]
There is no way to autogenerate from a resolver. You can use codegeneration on your schema to create the corresponding api :https://netflix.github.io/dgs/advanced/java-client/#type-safe-query-api

**Reply by Rfank2021** [reply]
I know the way to generate the query class and projection class, but I still need to specify fields in projection. Is there a way to specify all fields at once?
